examples/reference/elements/bokeh/Points.ipynb

```diff
@@ -56,8 +59,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Points (color='k' marker='+' size=10)\n",
-    "hv.Points(coords) + hv.Points(coords)[0.6:0.8,0.2:0.5]"
+    "layout = hv.Points(coords) + hv.Points(coords)[0.6:0.8,0.2:0.5]\n",
+    "layout.opts(opts.Points(color='k', marker='+', size=10))"
```

Comment:
I'd reuse the points defined in the previous cell here, no need to style at all here then.